media_player/alexa.py

```diff
@@ -12,6 +12,9 @@
 import requests
 import voluptuous as vol
 from bs4 import BeautifulSoup
+import pickle
+from threading import Lock
```

Comment:
Eventually imports not native to home assistant need to be be added to the requirements and loaded during the setup_platform phase. Although this will all be solved with a custom pip module eventually.

Reply:
My understanding is pickle is actually native to python3. Threading is gone.